src/components/Comments/WriteComment.js

```diff
@@ -194,8 +223,9 @@ export const styles = StyleSheet.create({
     paddingLeft: 6,
     paddingRight: 8,
     marginEnd: 8,
-    flex: 1
-  },
+    flex: 1,
+    height: isViewOnly ? 36 : undefined
+  }),
   btn: (isDisableSubmit) => ({
     backgroundColor: !isDisableSubmit ? COLORS.bondi_blue : COLORS.concrete,
     borderRadius: 18,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [216-228]

The `content` style now accepts a parameter `isViewOnly` and sets the height accordingly. This is a good way to adjust the style based on the state of the component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->